src/routes/event/index.tsx

```diff
@@ -3,11 +3,31 @@ import LoadData from '../../load-data'
 import Page from '../../components/page'
 import { getEventMatches, getEventInfo } from '../../api'
 import { MatchCard } from '../../components/match-card'
+import InfoGroupCard from '../../components/info-group-card'
+import {
+  mapMarker,
+  infoOutline,
+  calendar,
+  googleMaps,
+  video,
+  twitch,
+  youtube,
+} from '../../icons'
+import { formatDateRange } from '../../utils'
+import Icon from '../../components/icon'
+import Chip from '../../components/chip'
+import style from './style.css'
+import IconButton from '../../components/icon-button'
 
 interface Props {
   eventKey: string
 }
 
+const icons = { twitch, youtube }
+
+const gmapsUrl = (lat: number, lon: number) =>
+  `https://www.google.com/maps/?q=${lat},${lon}`
```

Comment:
I would if there was a URL that could give me that, but the URL you gave is in my opinion worse than the one I have already because it is not zoomed in and it doesn't have the pointer thing:

https://www.google.com/maps/place/Hershey+Centre/@43.632189,-79.653465,15z
vs
https://www.google.com/maps/?q=43.632189,-79.653465